docs/linter.md

```diff
@@ -8,13 +8,14 @@ and more.
 ## `ruff check`
 
 `ruff check` is the primary entrypoint to the Ruff linter. It accepts a list of files or
-directories, and lints all discovered Python files, optionally fixing any fixable errors:
+directories, and lints all discovered Python files (including those in subdirectories), optionally 
+fixing any fixable errors:
 
 ```console
-$ ruff check                  # Lint all files in the current directory.
-$ ruff check --fix            # Lint all files in the current directory, and fix any fixable errors.
-$ ruff check --watch          # Lint all files in the current directory, and re-lint on change.
-$ ruff check path/to/code/    # Lint all files in `path/to/code` (and any subdirectories).
+$ ruff check                  # Lint files in the current directory and any subdirectories.
+$ ruff check --fix            # Lint files in the current directory and any subdirectories and fix any fixable errors.
+$ ruff check --watch          # Lint files in the current directory and any subdirectories and re-lint on change.
+$ ruff check path/to/code/    # Lint files in `path/to/code` and any subdirectories.
 ```
```

Comment:
I can see how this is unclear on `main`, but the suggested rewording here _does_ feel a little repetitive now. What about something like this?

````
`ruff check` is the primary entrypoint to the Ruff linter. It accepts a list of files or
directories, and lints all discovered Python files, optionally fixing any fixable errors.
Python files are always searched for recursively in a given directory and all its
subdirectories:

```console
$ ruff check                  # Lint all files in the current directory.
$ ruff check --fix            # Lint all files in the current directory, and fix any fixable errors.
$ ruff check --watch          # Lint all files in the current directory, and re-lint on change.
$ ruff check path/to/code/    # Lint all files in `path/to/code`
```
````

Reply:
PS do you have opinions on:
`Lint all files in the current directory.`
vs.
`Lint files in the current directory.`

I prefer the latter since it hints there is some filtering without losing clarity, e.g. only Python files, respect .gitignore, etc.